### ISSUE TITLE
refactor pwd messages to only create pgpmime attachments once per batch

### ISSUE THREAD

**Issue by tomholub** [FlowCrypt/flowcrypt-browser#4495]
For a refactor in another issue:
 - this `sendablePwdMsg` method could be removed
 - `formatSendablePwdMsgs` could call `const emailIntroAndLinkBody = await this.formatPwdEncryptedMsgBodyLink(msgUrl);` and `    return await SendableMsg.createPwdMsg(this.acctEmail, this.headers(newMsg), emailIntroAndLinkBody, messageBodyAsPgpMimeAttachmentsEncryptedForSender,
      { isDraft: this.isDraft, externalId });`
 - `messageBodyAsPgpMimeAttachmentsEncryptedForSender` could be created only once, and reused in each cycle of the loop in `formatSendablePwdMsgs`
 - to create `messageBodyAsPgpMimeMetaAttachmentsEncryptedForSender` you could reuse most of existing `sendablePwdMsg` that relates to creating this pgpmime attachments, and this new method could be called `encryptMessageBodyAsPgpMimeMetaAttachments` and only sender pubkeys would be passed into it to create `messageBodyAsPgpMimeMetaAttachmentsEncryptedForSender`

The point is that this encryption only needs to happen once per batch, not once per each pwd msg, and it only needs to use pubkeys of sender, not everybody's pubkeys.

_Originally posted by @tomholub in https://github.com/FlowCrypt/flowcrypt-browser/pull/4284#discussion_r901425146_

**Comment by tomholub** [comment]
<img width="890" alt="image" src="https://user-images.githubusercontent.com/6306961/174568037-e6484b17-1f14-4551-b34d-cf1314a032a3.png">
